packages/now-cli/src/util/errors-ts.ts

```diff
@@ -1179,6 +1179,16 @@ export class ProjectNotFound extends NowError<'PROJECT_NOT_FOUND', {}> {
   }
 }
 
+export class ProjectUnauthorized extends NowError<'PROJECT_UNAUTHORIZED', {}> {
```

Comment:
I _think_ we're moving away from these Error subclasses in favor of just instantiating a `NowBuildError()` instance (from `@vercel/build-utils`). This way, the `message` and `code` and `link` are all self-contained in the error instance. @styfle can confirm.

Reply:
Indeed we can use NowBuildError() for CLI and Build errors. However, this error is coming from the API so we should actually have the API return those 3 properties instead and then use `output.prettyError(err)`.